Experimental: istio-router-check tool (#403)
getyourguide/istio-config-validator: cmd/istio-router-check/README.md

> # Istio Router Check

Reviewer: Would be good to link to this from the `README.md` at the root of the repository

Author: I'd prefer to keep it here and not advertise too much until is a bit more mature/tested internally.

An _experimental_ wrapper command for Envoy [Route Table Check Tool](https://www.envoyproxy.io/docs/envoy/latest/configuration/operations/tools/router_check).

> 1. It parses Istio configuration and generate it in the Envoy [HTTP Route](https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/route/v3/route_components.proto#http-route-components-proto) format required by the tool.

Reviewer: ```suggestion
1. It parses Istio configuration into the Envoy [HTTP Route](https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/route/v3/route_components.proto#http-route-components-proto) format required by the tool.
```
